### DISCUSSION TITLE
Adding support for trivy addtional findings (ex: Iac).

### DISCUSSION THREAD

**Discussion by rajitha-vk** [DependencyTrack/dependency-track#3919] [Ideas]
Trivy has extended features such as scanning IaC.
Given the mature portal and integrations of DT wouldn't it be a good match if DT has the ability to directly support trivy scan outputs or support the additional featuers through already available Trivy analyzer integration?

**Comment by larsriehn** [comment]
Another Trivy flag I'd like to see in DT is related to the "--ignore-unfixed" option of Trivy.
It would be perfect to see this in DT as a flag "fix existing". Together with a filter option for this flag.
